src/org/wojcieszko/homework/homework08/SecondValue.java

```diff
@@ -0,0 +1,32 @@
+package org.wojcieszko.homework.homework08;
+
+import org.junit.Test;
+
+public class SecondValue {
+
+    private int[] table;
+
+    public int[] getTable() {
+        return table;
+    }
+
+    public SecondValue(int[] table) {
+        this.table = table;
+    }
+
+    public int leastValue() {
+
+        int[] value = new int[2];
```

Comment:
Tu masz niejawną inicjalizację zerami. Czyli dla danych `int table[] = {-1, -2, -3, -4};` wyjdzie `0`.

Reply:
Wiem, że Ada miała ten sam problem, ale nie wiedziałem z czego to wynikło i nie wiem na czym się skończyło.